src/cpp/core/CMakeLists.txt

```diff
@@ -282,3 +281,35 @@ target_link_libraries(
    ${Boost_LIBRARIES}
    ${CORE_SYSTEM_LIBRARIES}
 )
+
+# define executable (for running unit tests)
+if (RSTUDIO_UNIT_TESTS_ENABLED)
+
+  file(GLOB_RECURSE CORE_TEST_FILES "*Tests.cpp")
+
+  add_executable(rstudio-core-bin
+     TestMain.cpp
+     ${CORE_TEST_FILES}
+     ${CORE_HEADER_FILES}
+   )
+
+   set_target_properties(rstudio-core-bin
+      PROPERTIES
+         OUTPUT_NAME rstudio-core
+   )
+
+   target_link_libraries(rstudio-core-bin
+      rstudio-core
+      rstudio-core-synctex
+      rstudio-core-hunspell
+      ${Boost_LIBRARIES}
+      ${CORE_SYSTEM_LIBRARIES}
+   )
+   
+   # run tests after build
```

Comment:
Hmm, good point. It would be best if we could automate the running / reporting of unit tests though -- maybe we can talk a bit about that tomorrow?